app/js/containers/Artifacts.jsx

```diff
@@ -30,27 +42,28 @@ const ArtifactsContainer = ({ artifacts, onArtifactClick }) => {
                             </tr>
                         </thead>
                         <tbody>
-                            { Object.keys(artifacts).map(id =>
+                            { artifacts.map((artifact, index) =>
```

Comment:
Throw in the parens here as if they were curly-braces. It's a nice style. (Might even be worth seeing if we can get ESLint to notice)